internal/kubernetes/client.go

```diff
@@ -50,23 +51,25 @@ func NewClient(
 	}
 }
 
+// GetIngresses returns a list of ingresses in the cluster, this is done by using the jsonpath format option
+// of kubectl to get a valid json output to be unmarshalled.
 func (c *Client) GetIngresses() ([]Ingress, error) {
 	var result []Ingress
 
-	log, err := c.kubeRunner.Get("all", "ingress", "-o",
+	cmdOut, err := c.kubeRunner.Get("all", "ingress", "-o",
 		"jsonpath='[{range .items[*]}{\"{\"}\"Name\": \"{.metadata.name}\", "+
 			"\"Host\": [{range .spec.rules[*]}\"{.host}\",{end}]{\"}\"},{end}]'")
 	if err != nil {
 		return result, fmt.Errorf("error while reading resources from cluster: %w", err)
 	}
 
-	logStringIndex := logRegex.FindStringIndex(log)
+	logStringIndex := cmdOutRegex.FindStringIndex(cmdOut)
```

Comment:
I'd suggest renaming `logStringIndex` to something like `singlequoteIndex` to be consistent